internal/git/git.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/speakeasy-api/sdk-generation-action/internal/logging"
 	"github.com/speakeasy-api/sdk-generation-action/pkg/releases"
 
-	"github.com/google/go-github/v48/github"
+	"github.com/google/go-github/v54/github"
```

Comment:
Do we need to confirm that all our other requests still work? I can't imagine they wouldn't but might be wise to just run through a quick SDK setup in `dev`

Reply:
works fine: https://github.com/speakeasy-sdks-dev/test-petstore